### PR TITLE
Fix formatting with multiple dates and fix measure column header classes

### PR DESCRIPTION
**Pull request by patrikbraborec**
---

Supported PR commands:

| Command                | Description            |
| ---------------------- | ---------------------- |
| `ok to test`           | Re-run standard checks |
| `extended test`        | BackstopJS tests       |
| `extended check sonar` | SonarQube tests        |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

File: libs/sdk-backend-bear/src/convertors/dateFormatting/dateFormatter.ts [hunk 1]
```diff
@@ -10,37 +10,37 @@ import { createDateValueFormatter } from "./dateValueFormatter";
...
-        !dateAttributeUri ||
+        !dateAttributeUris?.length ||
         !dateFormat ||
         dateFormat === DEFAULT_DATE_FORMAT
     ) {
         return resultHeader;
     }
     const resultHeaderUri = resultHeader.attributeHeaderItem.uri;
-    if (!resultHeaderUri.startsWith(dateAttributeUri)) {
+    const foundUri = dateAttributeUris.find((dateAttributeUri) =>
```
Comment (no23reason) [inline libs/sdk-backend-bear/src/convertors/dateFormatting/dateFormatter.ts:40]: Nitpick: `some` might be more appropriate here, you never use the actual value, you only check if any element meets the criteria.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

File: libs/sdk-backend-bear/src/convertors/dateFormatting/dateFormatter.ts (hunk 1 above)
Comment (patrikbraborec) [reply libs/sdk-backend-bear/src/convertors/dateFormatting/dateFormatter.ts]: Done

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/2355/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/3980/ : FAILURE in 15m 19s

**Comment by patrikbraborec** [comment]
extended check sonar

**Comment by yenkins** [comment]
### Sonar scan result
 - No new bug found. Congratulation!

More detail, see in https://sonarqube-dev.intgdc.com/dashboard?id=gooddata-ui-sdk-check-extended-sonar-PR1009

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/2364/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-el7/78/ : SUCCESS in 7m 32s (non-voting)

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/2363/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/3985/ : SUCCESS in 13m 27s

**Comment by patrikbraborec** [comment]
extended test

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/2365/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/657/ : SUCCESS in 17m 01s

File: libs/sdk-backend-bear/src/convertors/dateFormatting/dateFormatter.ts [hunk 2]
```diff
@@ -10,37 +10,37 @@ import { createDateValueFormatter } from "./dateValueFormatter";
 import { DateFormat, DEFAULT_DATE_FORMAT } from "./dateValueParser";
 import { createDefaultDateFormatter } from "./defaultDateFormatter";
 
-export function findDateAttributeUri(dimensions: IDimensionDescriptor[]): string | undefined {
-    const dateAttrDescriptor = dimensions
+export function findDateAttributeUris(dimensions: IDimensionDescriptor[]): string[] {
```
Comment (lupko) [inline libs/sdk-backend-bear/src/convertors/dateFormatting/dateFormatter.ts:13]: please mark this fun as TODO: rewrite. move search for date attributes into data view facade / meta() section

Comment (patrikbraborec) [reply libs/sdk-backend-bear/src/convertors/dateFormatting/dateFormatter.ts:14]: done

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/2366/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/3986/ : SUCCESS in 16m 09s

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by yenkins** [comment]
### Sonar scan result
 - No new bug found. Congratulation!

More detail, see in https://sonarqube-dev.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR1009

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/722/ : SUCCESS in 32s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/985/ : SUCCESS in 18s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/3987/ : SUCCESS in 16m 02s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-el7/79/ : SUCCESS in 9m 30s (non-voting)

**Pull request merged**